examples/asr/quantization/speech_to_text_quant_infer.py

```diff
@@ -113,7 +113,9 @@ def main():
                         module.disable()
 
     labels_map = dict([(i, asr_model.decoder.vocabulary[i]) for i in range(len(asr_model.decoder.vocabulary))])
-    wer = WER(vocabulary=asr_model.decoder.vocabulary, use_cer=args.use_cer)
+    decoding_cfg = CTCCharDecodingConfig()
+    char_decoding = CTCCharDecoding(decoding_cfg, vocabulary=labels_map)
```

Comment:
Is this script supposed to just work with char-based models?
We don't have so many char-based models anymore. How about we raise an error when BPE models are used or lat least at comments on the top?

Reply:
This quantization script mostly works with Quartznet and has not been updatedin a very long time. Its mostly for Quartznet nowadays.